packages/firebase_crashlytics/test/firebase_crashlytics_test.dart

```diff
@@ -0,0 +1,98 @@
+import 'package:firebase_crashlytics/firebase_crashlytics.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('$Crashlytics', () {
+    final List<MethodCall> log = <MethodCall>[];
+
+    final Crashlytics crashlytics = Crashlytics.instance;
+
+    setUp(() async {
+      Crashlytics.channel
+          .setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+        switch (methodCall.method) {
+          case 'Crashlytics#isDebuggable':
+            return true;
+          case 'Crashlytics#setUserEmail':
+            return true;
+          case 'Crashlytics#setUserIdentifier':
+            return true;
+          case 'Crashlytics#setUserName':
+            return true;
+        }
+      });
+      log.clear();
+    });
+
+    test('onError', () async {
+      final FlutterErrorDetails details = FlutterErrorDetails(
+        exception: 'foo exception',
+        stack: StackTrace.current,
+        library: 'foo library',
+        context: 'foo context',
+        informationCollector: (StringBuffer information) {
+          information.writeln('foo information');
+        },
+      );
+      crashlytics.enableInDevMode = true;
+      crashlytics.log('foo');
+      await crashlytics.onError(details);
+      expect(log[0].method, 'Crashlytics#onError');
+      expect(log[0].arguments['exception'], 'foo exception');
```

Comment:
I think it would be more readable and consistent with other tests to write this as a single expect comparing log with an array of maps.

Reply:
It sounds like StackTrace.current might make it hard to use the isMethodCall matcher, in which case this is fine.

Reply:
After chat with @collinjackson we will leave this as is, since `Stacktrace.current` will likely not be the same on each execution.